BadHabbit/BadHabbit/Kit/DesignKit/Models/LayoutConstants.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LayoutConstants.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 25/07/2023.
+//
+
+import Foundation
+
+enum LayoutConstants {
+    enum ActionButton {
+        static let height = 50
+        static let leadingInset = 16
+        static let trailingInset = 16
+        static let bottomInset = 20
+        static let topOffset = 30
```

Comment:
This enum's defines global LayoutConstants. In other words constants that used on another screens for such ActionButton. Does topOffset will be reused for another ActionButton in most of the cases?

Reply:
No, it doesn't.  topOffset was removed from enum. 
Finally I've got why our enum LayoutConstants look like that :confounded:
Thank you!